nav2_bringup/params/nav2_params.yaml

```diff
@@ -211,7 +211,7 @@ local_costmap:
         mark_threshold: 0
         observation_sources: scan
         scan:
-          topic: /scan
+          topic: scan
```

Comment:
Doesn't this try to find the node's relative topic? `global_costmap/global_costmap/scan`?